packages/blob/src/list.ts

```diff
@@ -64,6 +83,10 @@ export async function list(
   if (options?.cursor) {
     listApiUrl.searchParams.set('cursor', options.cursor);
   }
+  if (options?.mode) {
+    listApiUrl.searchParams.set('mode', options.mode);
```

Comment:
Can we add one unit test to check the new option is passed down to the API? we already have list tests